platform/android/export/export.cpp

```diff
@@ -1341,6 +1356,42 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return str;
 		}
 	}
+
+	// Creates strings.xml files inside the gradle project for different locales.
+	void _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset) {
+		XMLParser parser;
+		const char *locales[] = {
+			"ar", "bg", "ca", "cs", "da", "de", "el", "en", "es", "es_ES", "fa", "fi", "fr", "hi",
+			"hr", "hu", "in", "it", "iw", "ja", "ko", "lt", "lv", "nb", "nl", "pl", "pt", "ro",
+			"ru", "sk", "sl", "sr", "sv", "th", "tl", "tr", "uk", "vi", "zh", "zh_HK", "zh_TW"
+		};
```

Comment:
What's the reference for this list? Is it a fixed list of locales supported on Android?

It's surprising to see `es` + `es_ES`, but no distinction for `pt_BR` and `pt_PT`.

Reply:
This list of locales is taken directly from the strings.xml files available in [this directory](https://github.com/amanj120/godot/tree/3.2/platform/android/java/lib/res)

Reply:
If there are more locales you think are necessary, please let me know, and I can add them.

Reply:
Well is there any Android reference that describes what locales should be handled?

Reply:
Also since this list doesn't change, it should be a constant outside of the method.

Reply:
There are a lot of differing sources on the full locale list for Android, none of them seem to be fully encompassing. This is a list from the [Google Play Console Help Page](https://support.google.com/googleplay/android-developer/table/4419860?hl=en). This is a [stackoverflow page](https://stackoverflow.com/questions/7973023/what-is-the-list-of-supported-languages-locales-on-android) about the same topic, but it has many more locales listed.

Reply:
I changed the logic here a little bit. Instead of having a hardcoded list of locales, what happens now is that this function looks inside the `res://android/build/res` directory for all directories that start with `"values-"`. This means that if at any point in the future, Godot decides to support additional locales, the only change that needs to be made is adding the `platform/android/java/app/res/values-<new locale>/godot_project_name_string.xml` file for the new locale.